### PR TITLE
Reusable form

### PR DESCRIPTION
**Pull request by polo13999**
use adopt & context  to build

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,16 @@
...
+
+## Getting Started
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes.
+
+### Prerequisites
+
+This project runs on [Next](https://github.com/zeit/next.js/), and [Ant Design](https://ant.design/docs/react/introduce)and [node](https://nodejs.org/en/)
+
+=======
```
Comment (LIYINGZHEN) [inline README.md]: remove

File: README.md [hunk 2]
```diff
@@ -22,7 +33,8 @@ These instructions will get you a copy of the project up and running on your loc
 
 This project runs on [Next](https://github.com/zeit/next.js/), and [Ant Design](https://ant.design/docs/react/introduce)and [node](https://nodejs.org/en/)
 
-After installation, run `node -v` to make sure node installed correctly. Example
+> > > > > > > master
```
Comment (LIYINGZHEN) [inline README.md]: remove

File: README.md [hunk 3]
```diff
@@ -76,17 +88,11 @@ Run the migrations:
 Run the app:
 `$ docker-compose up -d`
 
-## Roadmap
```
Comment (LIYINGZHEN) [inline README.md]: use this

File: README.md [hunk 4]
```diff
@@ -76,17 +88,11 @@ Run the migrations:
...
-
-[Roadmap](./Learn.md)
-
 ## Authors
 
-* Polo Chi - [polo13999](https://github.com/polo13999)
```
Comment (LIYINGZHEN) [inline README.md]: use this

File: src/components/customer/form.js [hunk 5]
```diff
@@ -1,91 +0,0 @@
-import React, { Component } from 'react'
```
Comment (LIYINGZHEN) [inline src/components/customer/form.js]: Recover

File: src/components/form/index.js [hunk 6]
```diff
@@ -1,72 +0,0 @@
-import React, { Component } from 'react'
```
Comment (LIYINGZHEN) [inline src/components/form/index.js]: Recover

File: src/components/border/grpahql.js [hunk 7]
```diff
@@ -0,0 +1,34 @@
+import * as R from 'ramda'
```
Comment (LIYINGZHEN) [inline src/components/border/grpahql.js]: Ramda is not used.

**Comment by polo13999** [comment]
I fix again check it out

**Pull request merged**